### DISCUSSION TITLE
Our Windows and macOS nightly builds now contain SQLite 3.41.0

### DISCUSSION THREAD

**Discussion by justinclift** [sqlitebrowser/sqlitebrowser#3291] [Show and tell]
Just an FYI, our Windows and macOS nightly builds have been updated to contain the recently released SQLite 3.41.0:

&nbsp; &nbsp; https://nightlies.sqlitebrowser.org/latest/

**Comment by justinclift** [comment]
The automatically generated Linux AppImage builds now contain SQLite 3.41.0 too. :smile:

&nbsp; &nbsp; https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/continuous

**Comment by justinclift** [comment]
@deepsidhu1313 How easy would it be to update the Snap to SQLite 3.41.0? :smile:

**Comment by deepsidhu1313** [comment]
I will try this weekend

**Reply by deepsidhu1313** [reply]
i have upgraded the recipe to build sqlite from 3.41 branch. Sqlitebrowser on nighly ppa should have sqlite 3.41. Snap builds are failing and Launchpad team is looking into it. So whenever they resolve the infrastructure issue and retrigger the builds, snaps should also have sqlite 3.41.

**Reply by justinclift** [reply]
Awesome, good stuff @deepsidhu1313. :smile:
